### ISSUE TITLE
BitmapFont won't read in GWT with gdx 1.9.5

### ISSUE THREAD

**Issue by brownnrl** [libgdx/libgdx#4486]
#### Issue details

I've struggled to get BitmapFonts to read and load with GWT.  I believe it may be an issue with libgdx, as I've read a few different articles and gdx forum posts to the right approach for instantiating and rendering ```BitmapFont``` objects in GDX and tried many different variations on the recommendations.  I apologize if this is either a known issue or not the recommended usage.  Thank you for the great library.

#### Reproduction steps/code
* Run latest (as of issue date) gdxsetup.jar (GDX 1.9.5) with default settings with desktop/html selected.
* In ```MyGdxGame```, add a field for a ```BitmapFont font``` and add the following to the ```create``` method:

```java
                Gdx.app.setLogLevel(Application.LOG_DEBUG);
                Gdx.app.debug("Test", Gdx.files.internal("arial-15.fnt").readString());
                font = new BitmapFont(
                        Gdx.files.internal("arial-15.fnt"),
                        Gdx.files.internal("arial-15.png"), false, true)
```

* Copy the arial-15.fnt and arial-15.png files out of the gdx lib into the assets folder.
* Set a working directory to run from the assets folder, see the ```DesktopLauncher.main``` and see that it executes correctly, showing the font rendered and the contents of arial-15.fnt displayed in the console.

![screenshot from 2016-12-14 13-23-21](https://cloud.githubusercontent.com/assets/672163/21195129/83737af2-c200-11e6-9523-20c71cc5a191.png)

* run ```gradlew html:dist```, used python SimpleHTTPServer to serve from the ```html/build/dist``` directory
* get a caught exception about inability to read the ```BitmapFont```.

#### Version of LibGDX and/or relevant dependencies

GDX 1.9.5 with all defaults versions set from the gdxsetup.jar.  I've tried with Java 1.7 and 1.8, both oracle and openjdk on windows 7, centos, and ubuntu.  Results copied were with OpenJDK Runtime Environment (build 1.8.0_31-b13) and centos.

#### Stacktrace
You get the following caught exception, though the web server log indicates the font files served up and the log content showing the file *.fnt contents were successfully read.

![screenshot from 2016-12-14 13-16-21](https://cloud.githubusercontent.com/assets/672163/21194907/b0b55e96-c1ff-11e6-911b-0b77d10c4b62.png)

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [X] HTML/GWT
- [X] Windows
- [X] Linux
- [ ] MacOS

**Comment by MobiDevelop** [comment]
This has been fixed already. We'll get a new release out soon with the fix.

**Comment by badlogic** [comment]
This weekend.

**Comment by brownnrl** [comment]
Thank you, sorry I did a check through the issues to see if it was a duplicate.  Must have been to cursory.  Thanks again for all your effort.

**Comment by brownnrl** [comment]
I apologize for pinging a dead thread, but I was just curious if there was a bug release coming for this issue?  Was hoping to just bump the gradle version pointer up instead of applying the fix manually, but I still have a bit of time before it's needed.  Thank you again.

**Comment by BenMcLean** [comment]
This seriously needs a bug fix release. I can't develop on snapshot if I'm doing regular GWT testing.

**Comment by BenMcLean** [comment]
This appears to be fixed in 1.9.6 :)
